unix/vncserver/selinux/vncsession.te

```diff
@@ -81,4 +81,7 @@ optional_policy(`
 	')
 	userdom_admin_home_dir_filetrans(userdomain, vnc_home_t, dir, ".vnc")
 	userdom_user_home_dir_filetrans(userdomain, vnc_home_t, dir, ".vnc")
+
+	gnome_config_filetrans(vnc_session_t, vnc_home_t, dir, "tigervnc")
+	gnome_data_filetrans(userdomain, vnc_home_t, dir, "tigervnc")
```

Comment:
We also need to add permissions to create `~/.local`, `~/.local/state` and `~/.local/state/tigervnc`. Unfortunately, I cannot find the proper interfaces for it. There is one for `~/.cache` (which GDM needs), so I suspect there simply isn't an interface.

@zpytela ?

`gnome_filetrans_home_content()` seems to be part of it, as it sets up the transition. But I assume we also need `gnome_filetrans_admin_home_content()`. But the compilation breaks if I add both. :/